packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -68,6 +68,8 @@ type Value<'scope> = v8::Local<'scope, v8::Value>;
 type Function<'scope> = v8::Local<'scope, v8::Function>;
 type Array<'scope> = v8::Local<'scope, v8::Array>;
 
+const MB: usize = 1_000_000;
```

Comment:
Can we define the defaults here? Then it's easier to find them later.

```suggestion
const MB: usize = 1_000_000;

// Default heap limits used for each `v8::Isolate` in MB
const DEFAULT_INITIAL_HEAP_CONSTRAINT_MB = 0;
const DEFAULT_MAX_HEAP_SIZE_MB = 4_000;
```

Reply:
Removed in https://github.com/hashintel/hash/pull/446/commits/63d7cb7d1d40fc863a04e438dc24fa84a7721205